### PR TITLE
Add OpenRouteService factory

### PR DESCRIPTION
**Pull request by bezin**
This adds a factory for the OpenRouteService provider.

### CODE REVIEW

File: ProviderFactory/OpenRouteServiceFactory.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    protected static function configureOptionResolver(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'httplug_client' => null,
+            'api_key' => null,
+        ]);
+
+        $resolver->setAllowedTypes('httplug_client', [HttpClient::class, 'null']);
+        $resolver->setAllowedTypes('api_key', ['string']);
```
Comment (norkunas) [inline ProviderFactory/OpenRouteServiceFactory.php:45]: As `api_key` is required in the `OpenRouteService`, we should configure it as required :)

Comment (bezin) [reply ProviderFactory/OpenRouteServiceFactory.php:46]: Good catch, thanks! I required it in [f5c6db4](https://github.com/geocoder-php/BazingaGeocoderBundle/pull/329/commits/f5c6db4b4ae4b019887279f37821f674e34e235f)

**Review by norkunas — APPROVED** [review]

**Comment by norkunas** [comment]
Thank you @bezin

**Comment by bezin** [comment]
Thanks for merging so quickly 👍

**Pull request merged**